shopelectro/urls.py

```diff
@@ -103,9 +103,10 @@ def cache_page(arg):  # Ignore PyFlakesBear
     custom_page_url(r'^(?P<page>robots\.txt)$', RobotsView.as_view()),
     custom_page_url(r'^(?P<page>search)/$', views.Search.as_view()),
     custom_page_url(r'^(?P<page>catalog)/$', cached_2h(views.CategoryTree.as_view())),
-    custom_page_url(r'^shop/(?P<page>order)/$', views.OrderPage.as_view()),
-    custom_page_url(r'^shop/(?P<page>order-success)/$', views.OrderSuccess.as_view()),
     custom_page_url(r'^(?P<page>sitemap)/$', SitemapPage.as_view()),
+    # this pages should show only actual state
+    custom_page_url(r'^shop/(?P<page>order)/$', never_cache(views.OrderPage.as_view())),
+    custom_page_url(r'^shop/(?P<page>order-success)/$', views.OrderSuccess.as_view()),
```

Comment:
we render some user data on the success page. So, let's not cache it too

![image](https://user-images.githubusercontent.com/3526796/54589944-14da8500-4a38-11e9-8c08-776aca631a35.png)